pages/how-to-contribute.md

```diff
@@ -0,0 +1,149 @@
+---
+title: How To Contribute
+permalink: /how-to-contribute/
+redirect_from: /contributor/
+---
+
+Contributions are welcome to all Apache Accumulo repositories. While most contributions are code,
+there are other ways to contribute to Accumulo:
+
+* answer questions on mailing lists
+* review pull requests
+* verify and test new releases
+* update the Accumulo website and documentation
+
+Contributions are reviewed (via GitHub pull requests) by
+the community before being merged by a committer.
+
+This document provides basic instructions for contributing to Accumulo.  If you are looking for more information, check out the more comprehensive [contributor guide](/contributors-guide/).
+
+## Issues
+
+Any contribution should have a corresponding issue. Accumulo uses [JIRA] for issue tracking. Before creating an issue,
+you will need to create an [Apache JIRA account][jira-signup]. If you need help finding an issue to work on, check out
+the [open issues labeled for newbies][newbie-issues] or [contact us][contact].
+
+## Repositories
+
+Contributions can be made to the following repositories. While the general contribution workflow is
+described below, repositories have special instructions in their `CONTRIBUTING.md` file which can be
+viewed by clicking on the `C` in the Links column below.
+
+| Repository                      | Links    | Description
+| ------------------------------- | -------- | -----------
+| [Accumulo][a]                   | [C][ac]  | Core Project
+| [Accumulo Website][w]           | [C][wc]  | Source for this website
+| [Accumulo Examples][e]          | [C][ec]  | Accumulo example code
+| [Accumulo Testing][t]           | [C][tc]  | Accumulo test suites such as continuous ingest and random walk
+| [Accumulo Docker][d]            | [C][dc]  | Source for Accumulo Docker image
+| [Accumulo Wikisearch][s]        | [C][sc]  | Accumulo example application that indexes and queries Wikipedia data
```

Comment:
+1 for listing all the repos, that is something we really need.  Do we need links to all of these contributing.md docs?  They all seem the same.  Unless, the idea is that they all have different procedures?  I would think that it would be the opposite... the contributing.md files at each repo would give a short description and then link to this page.

Reply:
They have different procedures so I think it's necessary.  GitHub links to these files when users create a PR so they are important.